pkgs/tools/misc/pastebinit/default.nix

```diff
@@ -21,11 +21,9 @@ stdenv.mkDerivation rec {
 
   patches = [
     # Required to allow pastebinit 1.5 to run on Python 3.8
-    (fetchpatch {
-      name = "use-distro-module.patch";
-      url = "https://bazaar.launchpad.net/~arnouten/pastebinit/python38/diff/264?context=3";
-      sha256 = "1gp5inp4xald65xbb7fc5aqq5s2fhw464niwjjja9anqyp3zhawj";
-    })
+    ./use-distro-module.patch
```

Comment:
I see this patch needed to be updated, thanks for doing that. Have you also submitted this improvement upstream?